### PR TITLE
Maximum square matrix size Python code

### PR DESCRIPTION
**Pull request by rutujadhanawade**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

I have added python code of Maximum Square Matrix with all 1s .
I have formatted code with PEP 8 guidelines.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#939

### CODE REVIEW

**Comment by rutujadhanawade** [comment]
@iamrajiv @Kajol-Kumari @plazzy99 @atarax665  Please review this its been 6 days since I have created this PR.

File: Python/other/Maximum_Square_Matrix.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# find the size of Maximum Square Matrix With All 1s
```
Comment (iamrajiv) [inline Python/other/Maximum_Square_Matrix.py:1]: @rutujadhanawade Format your code according to PEP 8 rules. You can refer to this http://pep8online.com/.

File: Python/README.md [hunk 2]
```diff
@@ -137,6 +137,8 @@ _add list here_
 - [Merging Sorted arrays](other/merge_sorted_arrays.py)
```
Comment (iamrajiv) [inline Python/README.md:137]: @rutujadhanawade Just add `- [Maximum Square Matrix with all 1's](other/Maximum_Square_Matrix.py)` in README in alphabetical order. Do not edit other links.

**Review by iamrajiv — CHANGES_REQUESTED** [review]

File: Python/other/Maximum_Square_Matrix.py (hunk 1 above)
Comment (rutujadhanawade) [reply Python/other/Maximum_Square_Matrix.py:1]: I have already formatted it. @iamrajiv 
python

File: Python/README.md (hunk 2 above)
Comment (rutujadhanawade) [reply Python/README.md:137]: @iamrajiv done :+1:

File: Python/other/Maximum_Square_Matrix.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    # creating a null matrix
+    ans = 0
+    temp = [[0] * len(mat[0])] * len(mat)
+    # Traversing through the given matrix
+    for i in range(len(mat)):
+        for j in range(len(mat[0])):
+            temp[i][j] = mat[i][j]
+            # checking if it is not a corner case:
+            if i > 0 and j > 0 and mat[i][j] == 1:
+                temp[i][j] = min(temp[i][j - 1], temp[i - 1][j], temp[i - 1][j - 1])
```
Comment (iamrajiv) [inline Python/other/Maximum_Square_Matrix.py]: @rutujadhanawade line too long (84 > 79 characters)

**Review by iamrajiv — CHANGES_REQUESTED** [review]

File: Python/other/Maximum_Square_Matrix.py (hunk 1 above)
Comment (rutujadhanawade) [reply Python/other/Maximum_Square_Matrix.py:1]: Done . Please have a look at it.

File: Python/other/Maximum_Square_Matrix.py (hunk 3 above)
Comment (rutujadhanawade) [reply Python/other/Maximum_Square_Matrix.py]: Done . Please have a look at it.

**Review by iamrajiv — APPROVED** [review]

**Review by Kajol-Kumari — APPROVED** [review]
LGTM :+1:

**Review by vatsalkesarwani12 — APPROVED** [review]
👍🏻

**Pull request merged**